docs/guides/capability-sets.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Session Capability Sets For Common Testing Scenarious
+---
+
+This article describes necessary capabilities that must be provided in order
+to implement some common automation testing scenarios.
+It only describes very minimum sets of capabilities required to
+be included. For refined setups more of them might need to be provided. Check the
+[Capabilities](../reference/capabilities.md) article for more details
+on each option available for the fine-tuning of XCUITest driver sessions.
+
+### Simple iOS App Test On a Real Device
```

Comment:
This and the other header titles could be made shorter and to the point. How about these:
* Application File (Real Device)
* Application File (Simulator)
* Safari (Real Device)
* Safari (Simulator)
* Pre-Installed App (Real Device)
* Pre-Installed App (Simulator)
* Deeplink (Real Device running iOS 17+)
* Deeplink (Simulator running iOS 17+)
* Custom Launch (Real Device)
* Custom Launch (Simulator)

Reply:
makes sense. Changed